packages/flutter/lib/src/material/theme.dart

```diff
@@ -161,6 +161,48 @@ class Theme extends StatelessWidget {
         .resolveFrom(context);
   }
 
+  /// Retrieves the [Brightness] to use for descendant widgets, based
+  /// on the value of [ThemeData.brightness] in the given [context].
+  ///
+  /// If no [InheritedTheme] can be found in the given [context], or its `brightness`
+  /// is null, it will fall back to [MediaQueryData.platformBrightness].
+  ///
+  /// Throws an exception if no valid [InheritedTheme] or [MediaQuery] widgets
+  /// exist in the ancestry tree.
+  ///
+  /// See also:
+  ///
+  /// * [maybeBrightnessOf], which returns null if no valid [InheritedTheme] or
+  ///   [MediaQuery] exists, instead of throwing.
+  /// * [ThemeData.brightness], the property takes precedence over
+  ///   [MediaQueryData.platformBrightness] for descendant Cupertino widgets.
+  static Brightness brightnessOf(BuildContext context) {
+    final _InheritedTheme? inheritedTheme =
+        context.dependOnInheritedWidgetOfExactType<_InheritedTheme>();
+    return inheritedTheme?.theme.data.brightness ?? MediaQuery.platformBrightnessOf(context);
+  }
+
+  /// Retrieves the [Brightness] to use for descendant Cupertino widgets, based
```

Comment:
Same here

```suggestion
  /// Retrieves the [Brightness] to use for descendant Material widgets, based
  /// on the value of [ThemeData.brightness] in the given [context].
  ///
  /// If no [InheritedTheme] can be found in the given [context], it will fall
  /// back to [MediaQueryData.platformBrightness].
  ///
  /// Returns null if no valid [InheritedTheme] or [MediaQuery] widgets exist in
  /// the ancestry tree.
  ///
  /// See also:
  ///
  /// * [ThemeData.brightness], the property that takes precedence over
  ///   [MediaQueryData.platformBrightness] for descendant Material widgets.
  /// * [brightnessOf], which throws if no valid [InheritedTheme] or
  ///   [MediaQuery] exists, instead of returning null.
```